crates/nu-protocol/src/lev_distance.rs

```diff
@@ -57,6 +53,11 @@ pub fn lev_distance(a: &str, b: &str, limit: usize) -> Option<usize> {
     }
 }
 
+/// Finds the Levenshtein distance between two strings.
+pub fn levenshtein_distance(a: &str, b: &str) -> usize {
+    lev_distance(a, b, usize::max_value())
+        .expect("It is impossible to exceed the supplied limit since all types involved are usize.")
```

Comment:
I think we shouldn't use `expect` here because Nushell should never panic. There might be a bug in `lev_distance()` that could cause it to return `None`. While annoying, it's better to avoid `expect` and return `Option<usize>` or use some default value (i.e. very high? I'm not sure how the algorithm works).

Reply:
that makes sense. By the same logic as the expect statement, we know for sure (if `lev_distance` is correctly implemented at least) that the value will be `usize::max_value()` or less, so I'll default to that if we get a `None`.

Reply:
I do think it's worth noting though that this `expect` is [already in the code](https://github.com/nushell/nushell/blob/517dc6d39e32e786b6fa0fa365c5397004a84c87/crates/nu-protocol/src/shell_error.rs#L1117), I just moved where the function was defined. I can remove it as I said above still, as long as the scope creep is not a concern.